### PR TITLE
[release-2.7] Manual sync of ManagedClusterAddon timeout bump

### PR DESCRIPTION
**Pull request by dhaiducek**
Manual cherry-pick:
- https://github.com/stolostron/governance-policy-addon-controller/pull/296

### CODE REVIEW

File: test/e2e/case2_config_deployment_test.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ var _ = Describe("Test config-policy-controller deployment", func() {
 
 			By(logPrefix +
 				"removing the config-policy-controller deployment when the ManagedClusterAddOn CR is removed")
-			Kubectl("delete", "-n", cluster.clusterName, "-f", case2ManagedClusterAddOnCR, "--timeout=30s")
+			Kubectl("delete", "-n", cluster.clusterName, "-f", case2ManagedClusterAddOnCR, "--timeout=90s")
 			deploy := GetWithTimeout(
 				cluster.clusterClient, gvrDeployment, case2DeploymentName, addonNamespace, false, 180,
```
Comment (gparvin) [inline test/e2e/case2_config_deployment_test.go:104]: I made progress locally by adjusting this timeout for the case2 failures which I see you are still running into.

```suggestion
				cluster.clusterClient, gvrDeployment, case2DeploymentName, addonNamespace, false, 360,
```

File: test/e2e/case2_config_deployment_test.go [hunk 2]
```diff
@@ -143,7 +143,7 @@ var _ = Describe("Test config-policy-controller deployment", func() {
 
 			By(logPrefix +
 				"removing the config-policy-controller deployment when the ManagedClusterAddOn CR is removed")
-			Kubectl("delete", "-n", cluster.clusterName, "-f", case2ManagedClusterAddOnCR, "--timeout=30s")
+			Kubectl("delete", "-n", cluster.clusterName, "-f", case2ManagedClusterAddOnCR, "--timeout=90s")
 			deploy = GetWithTimeout(
 				cluster.clusterClient, gvrDeployment, case2DeploymentName, addonNamespace, false, 180,
```
Comment (gparvin) [inline test/e2e/case2_config_deployment_test.go:148]: See previous comment... I updated most of the 180s in my local testing.  I have still hit some other timeout locally so will try your changes with mine to see if I can find any other suggestions.

**Review by gparvin — APPROVED** [review]
I see the tests passed this time!  Nice

**Comment by dhaiducek** [comment]
@gparvin The initial failure wasn't either of the reviewed lines, and this passed on a re-run. I think I might prefer to let these changes ride, especially since this is the 2.7 branch and might not receive many more updates before it's EOL.

**Pull request merged**